nixos/modules/services/misc/paperless.nix

```diff
@@ -181,6 +181,7 @@ in
       example = literalExpression ''
         {
           PAPERLESS_OCR_LANGUAGE = "deu+eng";
+          PAPERLESS_DBHOST = "/run/postgresql";
```

Comment:
While you're at it, can you remove the unneeded `literalExpression`?

Reply:
I thought that was exactly where `literalExpression` was supposed to be used. What makes you think otherwise?

Reply:
Ah, sorry, I meant the extra string wrapping (`''`) of the attrset.

Reply:
`literalexpression` is for nix expressions that that can't be pretty-printed like `literalExpression "pkgs.hello"`.

Reply:
Checked the manual for good measure and it says that it is only required for complex values, where some kind of evaluation needs to happen.